app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -171,6 +173,17 @@ class ItemListFragment : CommonFragment(), ItemListView {
         adapter.updateItems(itemList)
     }
 
+    override fun updateAccountProfile(profile: AccountViewModel) {
+        navView.menuHeader.displayName.text = profile.displayEmailName
+        view!!.navView.menuHeader.accountName.text = profile.accountName
+        Picasso.get()
+            .load(profile.avatarFromURL)
+            .placeholder(R.drawable.ic_avatar_placeholder)
+            .resize(R.attr.avatar_image_size, R.attr.avatar_image_size)
```

Comment:
Oooh. This might not work the way you expect it. `R.attr.avatar_image_size` is a generated id, rather than the value itself.

```kotlin
Picasso.with(context)
    .load(url)
    .placeholder(R.drawable.whatever)
    .resizeDimen(R.dimen.avatar_image_size, R.dimen.avatar_image_size)
```

(I realize now I suggested you might move this into a `attrs.xml` file, and now I remember it should be in a `dimens.xml` file. Apologies.)

Reply:
You may also want to use a `fit` method, but I haven't read those docs too closely.

Reply:
done